dev/ci/ci-basic-overlay.sh

```diff
@@ -195,8 +203,11 @@
 ########################################################################
 # Bignums
 ########################################################################
-# Latest commit on master as of Sep 27, 2018
-: "${bignums_CI_REF:=f1a63cb4cce9a79e4406275fbb0ee2269947f8d2}"
+# There is no 8.9 branch
+# Below is the latest commit (Feb 4, 2019) on master as of April 4, 2019
+# 0cd435d0a3c731605536c83d0a731c3fc336cce7 => fails (requires Cyclic63)
+# This is the latest working commit (Feb 1, 2019) on master as of April 4, 2019
+: "${bignums_CI_REF:=5a7ba3d02d8c095ce31fc5f0a0dbc118c58dd433}"
```

Comment:
You shouldn't care about Bignums. It is not updated but for compatibility fixes. Therefore, the any working commit for 8.9 would be equivalent feature-wise.

Reply:
There are some minor changes in CI files and gitignore in the version I picked, but I could also use the same version as 8.9.0, if you prefer.

Reply:
I don't actually care, I was just saying to not bother in general.